mk/config.mk

```diff
@@ -176,6 +176,26 @@ CFG_WITH_USER_TA ?= y
 # case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Support for loading user TAs from a special section in the TEE binary
+# Such TAs are available even before tee-supplicant is available (hence their
+# name), but note that many services exported to TAs may need tee-supplicant,
```

Comment:
Indeed, BUILTIN_TA better describes what they really are. If no one objects I'll rename everything once we're done with the review. Otherwise it will bee too complex to follow the changes.